cypress/e2e/upload/uploadKeywords.cy.ts

```diff
@@ -0,0 +1,47 @@
+import { SEARCH_RESULT_LIST_SELECTOR } from '../searchResults/selectors';
+
+import { UPLOAD_SELECTOR } from './selectors';
+
+describe('Uploads csv file with keywords', () => {
```

Comment:
Inconsistency in the tense (see https://github.com/nimblehq/ic-nextjs-prabesh-dahal/pull/139/files#diff-e94c54ef92426639020c1c806ce70e7a33df6d26c6d75fe8ebc520cb37da6119L5)

```suggestion
describe('Upload keyword CSV file', () => {
```

Reply:
Resolved in https://github.com/nimblehq/ic-nextjs-prabesh-dahal/pull/139/commits/39b974da7c9b5cb8dabd4d8302922f773d482b1c